Fix twitch wrong feed

diegorodriguezv opened the pull request:
The problem occurs because the feed is retrieved each time (it changes quite often) and it loses sync with the cached copy in the browser (html feed) and the data cache (js). This solution uses a querystring parameter so that the streams are returned based on the game name, not the index relative to the changing feed list.

blissland commented:
Thanks! Since you are currently a lot more active on this project than me I have added you as a collaborator so you have commit access to this repo. I have not done this before so let me know if it does not work.

diegorodriguezv commented:
Thanks for trusting me with one of your "babies" and for this amazing piece of software. I'll be checking the commit access soon.

diegorodriguezv commented:
Respected Mr. Blissland.

I hope this email finds you well.

I have updated BlissFlixx and plan on maintaining the project. I have
recently added a new release which adds support for python 3 and several
minor fixes.

I have also updated the project github pages web site (gh-pages branch).
You can see how it looks at: https://diegorodriguezv.github.io/blissflixx/

Currently, the About section of the project page at:
https://github.com/blissland/blissflixx shows the website at:
blissflixx.rocks but the domain name expired and it would be preferable to
host it at the github pages site: https://blissland.github.io/blissflixx/
but it currently redirects to the expired domain.

Could you be so kind as to remove the custom domain from the pages
configuration?

Also I have changed the project catchphrase:
Turn your Raspberry Pi into a ROKU like Media Streamer.
For something more... er... catchy:
Raspberry Pi + TV = Smart streaming experience
Since each passing day less people know what a roku is.

Could you change it as well?

Thank you very much.

On Wed, Jun 5, 2019 at 11:57 AM blissland ***@***.***> wrote:

> Thanks! Since you are currently a lot more active on this project than me
> I have added you as a collaborator so you have commit access to this repo.
> I have not done this before so let me know if it does not work.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/blissland/blissflixx/pull/56?email_source=notifications&email_token=AACOBVKRNRBFIHRTJPM47ELPY7V7NA5CNFSM4HT6IO2KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODXALHXI#issuecomment-499168221>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AACOBVIJY3VKT3JRKKROR53PY7V7NANCNFSM4HT6IO2A>
> .
>


-- 
"Computer science is not about computers any more than astronomy is about
telescopes." E.W. Dijkstra (1930-2002)

blissland commented:
Hi Diego,

Thank you so much for maintaining this project!

The updates & website look awesome!!

I have made the changes you requested. I could not work out how to change
the redirect
to the custom domain so I just pointed the website to
https://diegorodriguezv.github.io/blissflixx/

Let me know if this is okay.

Thank you!!




On Mon, 20 Jun 2022 at 07:35, diegorodriguezv ***@***.***>
wrote:

> Respected Mr. Blissland.
>
> I hope this email finds you well.
>
> I have updated BlissFlixx and plan on maintaining the project. I have
> recently added a new release which adds support for python 3 and several
> minor fixes.
>
> I have also updated the project github pages web site (gh-pages branch).
> You can see how it looks at: https://diegorodriguezv.github.io/blissflixx/
>
> Currently, the About section of the project page at:
> https://github.com/blissland/blissflixx shows the website at:
> blissflixx.rocks but the domain name expired and it would be preferable to
> host it at the github pages site: https://blissland.github.io/blissflixx/
> but it currently redirects to the expired domain.
>
> Could you be so kind as to remove the custom domain from the pages
> configuration?
>
> Also I have changed the project catchphrase:
> Turn your Raspberry Pi into a ROKU like Media Streamer.
> For something more... er... catchy:
> Raspberry Pi + TV = Smart streaming experience
> Since each passing day less people know what a roku is.
>
> Could you change it as well?
>
> Thank you very much.
>
> On Wed, Jun 5, 2019 at 11:57 AM blissland ***@***.***> wrote:
>
> > Thanks! Since you are currently a lot more active on this project than me
> > I have added you as a collaborator so you have commit access to this
> repo.
> > I have not done this before so let me know if it does not work.
> >
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/blissland/blissflixx/pull/56?email_source=notifications&email_token=AACOBVKRNRBFIHRTJPM47ELPY7V7NA5CNFSM4HT6IO2KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODXALHXI#issuecomment-499168221
> >,
> > or mute the thread
> > <
> https://github.com/notifications/unsubscribe-auth/AACOBVIJY3VKT3JRKKROR53PY7V7NANCNFSM4HT6IO2A
> >
> > .
> >
>
>
> --
> "Computer science is not about computers any more than astronomy is about
> telescopes." E.W. Dijkstra (1930-2002)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/blissland/blissflixx/pull/56#issuecomment-1160032978>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAN4EBKULGOUS2DXI2EBNTLVQAGMNANCNFSM4HT6IO2A>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***>
>

diegorodriguezv commented:
Excellent. Thanks.

On Mon, Jun 20, 2022 at 5:57 AM blissland ***@***.***> wrote:

> Hi Diego,
>
> Thank you so much for maintaining this project!
>
> The updates & website look awesome!!
>
> I have made the changes you requested. I could not work out how to change
> the redirect
> to the custom domain so I just pointed the website to
> https://diegorodriguezv.github.io/blissflixx/
>
> Let me know if this is okay.
>
> Thank you!!
>
>
>
>
> On Mon, 20 Jun 2022 at 07:35, diegorodriguezv ***@***.***>
> wrote:
>
> > Respected Mr. Blissland.
> >
> > I hope this email finds you well.
> >
> > I have updated BlissFlixx and plan on maintaining the project. I have
> > recently added a new release which adds support for python 3 and several
> > minor fixes.
> >
> > I have also updated the project github pages web site (gh-pages branch).
> > You can see how it looks at:
> https://diegorodriguezv.github.io/blissflixx/
> >
> > Currently, the About section of the project page at:
> > https://github.com/blissland/blissflixx shows the website at:
> > blissflixx.rocks but the domain name expired and it would be preferable
> to
> > host it at the github pages site:
> https://blissland.github.io/blissflixx/
> > but it currently redirects to the expired domain.
> >
> > Could you be so kind as to remove the custom domain from the pages
> > configuration?
> >
> > Also I have changed the project catchphrase:
> > Turn your Raspberry Pi into a ROKU like Media Streamer.
> > For something more... er... catchy:
> > Raspberry Pi + TV = Smart streaming experience
> > Since each passing day less people know what a roku is.
> >
> > Could you change it as well?
> >
> > Thank you very much.
> >
> > On Wed, Jun 5, 2019 at 11:57 AM blissland ***@***.***> wrote:
> >
> > > Thanks! Since you are currently a lot more active on this project than
> me
> > > I have added you as a collaborator so you have commit access to this
> > repo.
> > > I have not done this before so let me know if it does not work.
> > >
> > > —
> > > You are receiving this because you authored the thread.
> > > Reply to this email directly, view it on GitHub
> > > <
> >
> https://github.com/blissland/blissflixx/pull/56?email_source=notifications&email_token=AACOBVKRNRBFIHRTJPM47ELPY7V7NA5CNFSM4HT6IO2KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODXALHXI#issuecomment-499168221
> > >,
> > > or mute the thread
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/AACOBVIJY3VKT3JRKKROR53PY7V7NANCNFSM4HT6IO2A
> > >
> > > .
> > >
> >
> >
> > --
> > "Computer science is not about computers any more than astronomy is about
> > telescopes." E.W. Dijkstra (1930-2002)
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/blissland/blissflixx/pull/56#issuecomment-1160032978
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AAN4EBKULGOUS2DXI2EBNTLVQAGMNANCNFSM4HT6IO2A
> >
> > .
> > You are receiving this because you modified the open/close state.Message
> > ID: ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/blissland/blissflixx/pull/56#issuecomment-1160300768>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AACOBVPI3NTL6EIRHP6T5ITVQBFCBANCNFSM4HT6IO2A>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
"Computer science is not about computers any more than astronomy is about
telescopes." E.W. Dijkstra (1930-2002)
